src/lib/plugins/blocks.js

```diff
@@ -2,6 +2,8 @@ const Vec3 = require('vec3').Vec3
 
 module.exports.player = function (player, serv) {
   player.changeBlock = async (position, blockType, blockData) => {
+    if (Math.round(player.gameMode) === 1 && player.heldItem?.name.endsWith('_sword')) return
```

Comment:
why

Reply:
You can't break blocks with sword on creative mode, idk why gameMode has decimals